content/blog/introducing-python-sdk-for-dscc.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Introducing Python SDK for DSCC
+date: 2023-03-29T13:45:50.979Z
+priority: 0
+author: Anusha Y and Sijeesh Kattumunda
+authorimage: /img/Avatar1.svg
+disable: false
+---
+In my previous blog, I discussed how to generate a Software Development Kit (SDK) from the Data services cloud console (DSCC) Open API spec using a third-party tool called OpenAPI generator. Today, I am going to talk more in detail on the Python SDK generated using the tool. As we progress, we will discuss the implementation aspects, and then take up a use-case to better understand how to use the SDK. 
+Before we get into SDK, let’s understand the difference between an Application Programming Interface (API) and an SDK. 
+APIs are a list of functions libraries that are used to communicate with a web service (via secure HTTP), whereas an SDK is a development kit that facilitates the usage of these APIs. APIs enable any software developer to create business opportunities by leveraging the capabilities provided by the API to extend their application whereas the SDK facilitates the process for developers. An SDK is a collection of software tools and programs for a specific application platform that allows developers to manipulate the functions supported by the service. These can be considered as a wrapper on the top of the APIs, making the code consumable by the application.
+One thing to note is that, the user always gets access to the latest DSCC API version through the SDKs. How? The SDK is designed and deployed (using CI/CD pipelines such as Jenkins) in such a way that with every new release of the DSCC Open API spec, the SDKs get updated automatically. Thus, keeping it up-to date without any manual intervention. This also reduces time which is spent waiting for updates with newer features.
```

Comment:
..to the latest Data Services Cloud Console API version through the SDKs.

...every new release of the Data Services Cloud Console Open API spec, the SDKs...